src/main/java/sirius/biz/jobs/batch/file/ArchiveImportJob.java

```diff
@@ -110,7 +110,7 @@ protected Optional<ExtractedFile> fetchEntry(String fileName) throws IOException
         Enumeration<? extends ZipEntry> zipEntries = zipFile.entries();
         while (zipEntries.hasMoreElements() && taskContext.isActive()) {
             ZipEntry zipEntry = zipEntries.nextElement();
-            if(zipEntry.getName().contains("__MACOSX")) {
+            if (FILE_FILTER.stream().anyMatch(zipEntry.getName()::contains)) {
```

Comment:
Optional: `startsWith` would be a more precise search, as done in some other places.